inc/namespace.php

```diff
@@ -95,6 +113,14 @@ function bootstrap() {
 	add_filter( 'style_loader_src', __NAMESPACE__ . '\\real_url_path', -10, 2 );
 }
 
+/**
+ * Remove the RSS feed and redirect to the home page.
+ */
+function disable_feed() {
+	wp_redirect( home_url() );
+	die();
```

Comment:
Could we send a 404 instead? Potentially we could remove the rewrites entirely.

Reply:
I changed it, would this work?